From 81040fa8bdbc71426b4106cf70754b7c4021c09f Mon Sep 17 00:00:00 2001
From: Damien Challet <damien.challet@gmail.com>
Date: Wed, 10 Jan 2024 15:12:56 +0100
Subject: [PATCH] hwdb: add Teclast X98 Pro sensor info (#30859)

added Teclast X98 Pro

(cherry picked from commit 3a0fbeacea40a6e1ad8b7f718182a7fa5be011a0)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 1086940b78..0e39a6a566 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -1021,6 +1021,10 @@ sensor:modalias:acpi:KIOX000A*:dmi:bvnAmericanMegatrendsInc.:bvr5.011:bd11/03/20
 sensor:modalias:acpi:KIOX000A*:dmi:*:svnTECLAST:pnX98PlusII:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
 
+# Teclast X98 Pro
+sensor:modalias:acpi:BMA250E*:dmi:*:svnTECLAST:pnX98Pro:*
+ ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
+
 #########################################
 # Thundersoft
 #########################################
